src/ksyms.cpp

```diff
@@ -23,8 +42,66 @@ std::string Ksyms::resolve(uint64_t addr, bool show_offset)
   return symbol.str();
 }
 
-Ksyms::~Ksyms()
+#ifdef USE_BLAZESYM
```

Comment:
eh, I don't think we need to guard the creation of these functions for something we're fairly sure we're going to switch to.